packages/eas-cli/src/commands/branch/publish.ts

```diff
@@ -118,11 +118,13 @@ export default class BranchPublish extends Command {
     const assetSpinner = ora('Uploading assets...').start();
     try {
       const assets = collectAssets(inputDir!);
+      console.log('[debug] assets -', 'uploading these assets:', JSON.stringify(assets, null, 2));
```

Comment:
```suggestion
      Log.log('[debug] assets -', 'uploading these assets:', JSON.stringify(assets, null, 2));
```

Reply:
Actually did you mean to delete these?

Reply:
yeah, my bad, this shouldn't have been committed. Will pull this out!